clipper_admin/clipper_admin/deployers/mxnet.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import print_function, with_statement, absolute_import
+import shutil
+import mxnet as mx
+import logging
+import re
+import os
+import json
+
+from ..version import __version__
+from ..clipper_admin import ClipperException
+from .deployer_utils import save_python_function, serialize_object
+
+logger = logging.getLogger(__name__)
+
+MXNET_MODEL_RELATIVE_PATH = "mxnet_model"
+
+
+def create_endpoint(
+        clipper_conn,
+        name,
+        input_type,
+        func,
+        mxnet_model,
+        mxnet_data_shapes,
+        default_output="None",
+        version=1,
+        slo_micros=3000000,
+        labels=None,
+        registry=None,
+        base_image="clipper/mxnet-container:{}".format(__version__),
+        num_replicas=1):
+    """Registers an app and deploys the provided predict function with MXNet model as
+    a Clipper model.
+    Parameters
+    ----------
+    clipper_conn : :py:meth:`clipper_admin.ClipperConnection`
+        A ``ClipperConnection`` object connected to a running Clipper cluster.
+    name : str
+        The name to be assigned to both the registered application and deployed model.
+    input_type : str
+        The input_type to be associated with the registered app and deployed model.
+        One of "integers", "floats", "doubles", "bytes", or "strings".
+    func : function
+        The prediction function. Any state associated with the function will be
+        captured via closure capture and pickled with Cloudpickle.
+    mxnet_model : mxnet model object
+        The MXNet model to save.
+    mxnet_data_shapes : list(int)
+        List of integers representing the dimensions of data used for model prediction.
+        Required because loading serialized MXNet models involves binding, which requires
+        the shape of the data used to train the model.
+        https://mxnet.incubator.apache.org/api/python/module.html#mxnet.module.BaseModule.bind
+
+        NOTE: Clipper may provide the model with variable size input batches. Because MXNet can't
```

Comment:
This is great. Can you make it a separate Notes section (below the Parameters section) so that it gets formatted properly with pydoc. Check out this example: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Done